kube/services/jobs/usersync-job.yaml

```diff
@@ -93,7 +93,7 @@ spec:
             name: "projects"
       containers:
       - name: usersync
-        GEN3_FENCE_IMAGE
+        GEN3_FENCE_MCRYPT_IMAGE
```

Comment:
I think adding this, enforces all commons to have `fence_mcrypt` in their manifest and doesn't seem like a great idea

Reply:
Yeah... It will force every job to use the new image for the usersync job regardless of if they're using the latest Fence, which is going to change what every Prod env runs without them explicitly updating a manifest or anything. We shouldn't do this. It doesn't allow someone to test the update in a staging/preprod before deploying in prod. We want to only use this new image if they're deploying the new Fence as configured in the manifest. Maybe we need a new manifest entry

Reply:
I suggested adding a new `fence_mcrypt` image line in the manifest's versions, specifically for jobs where usersync requires this branch. The team would then need to manually update the usersync job in the AdminVM.

Need opinions.